File: core-java-modules/core-java-8-datetime/src/main/java/com/baeldung/datetime/UseDuration.java
package com.baeldung.datetime;

import java.time.Duration;
import java.time.LocalTime;

public class UseDuration {

    public LocalTime modifyDates(LocalTime localTime, Duration duration) {
        return localTime.plus(duration);
    }

    public Duration getDifferenceBetweenDates(LocalTime localTime1, LocalTime localTime2) {
        return Duration.between(localTime1, localTime2);
    }
}

File: core-kotlin-modules/core-kotlin-2/src/main/kotlin/com/baeldung/dates/datetime/UseDuration.kt
package com.baeldung.dates.datetime

import java.time.Duration
import java.time.LocalTime

class UseDuration {

    fun modifyDates(localTime: LocalTime, duration: Duration): LocalTime {
        return localTime.plus(duration)
    }

    fun getDifferenceBetweenDates(localTime1: LocalTime, localTime2: LocalTime): Duration {
        return Duration.between(localTime1, localTime2)
    }
}